beacon-chain/sync/backfill/batch.go

```diff
@@ -68,13 +72,14 @@ type batch struct {
 	retryAfter     time.Time
 	begin          primitives.Slot
 	end            primitives.Slot // half-open interval, [begin, end), ie >= start, < end.
-	results        verifiedROBlocks
+	blocks         verifiedROBlocks
 	err            error
 	state          batchState
-	busy           peer.ID
+	peer           peer.ID
+	nextReqCols    []uint64
 	blockPid       peer.ID
```

Comment:
Oh actually this this the peer for block. So yes a comment about `peer` is useful.